cmake/modules/BuildJaeger.cmake

```diff
@@ -50,13 +50,15 @@ function(build_jaeger)
 			-Dthrift_HOME=${CMAKE_BINARY_DIR}/external
 			-DOpenTracing_HOME=${CMAKE_BINARY_DIR}/external)
 
-  set(dependencies opentracing thrift)
+  # build these libraries along with jaeger
+  set(dependencies ${dependencies} opentracing)
```

Comment:
i think `${dependencies}` is always empty here. might want to reference https://cmake.org/cmake/help/latest/command/set.html to understand the variable scope concept in CMake.